examples/utils/example-utils/src/assert.ts

```diff
@@ -0,0 +1,58 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * A browser friendly assert library.
+ *
+ * @param condition - The condition that should be true, if the condition is false an error will be thrown.
+ * Only use this API when `false` indicates a logic error in the problem and thus a bug that should be fixed.
+ * @param message - The message to include in the error when the condition does not hold.
+ *
+ * @remarks
+ * Use this instead of the node's 'assert' package, which needs a polyfill for web and has a big impact on bundle sizes.
+ * Can be used to narrow TypeScript types, allowing cases `assert(a instanceof A)` or `assert(typeof b === "number")` to replace `as` casts.
+ *
+ * See also {@link fail}.
```

Comment:
TSDoc / API-Extractor also support a `@see` tags whose semantics are basically "See also...". So this could be `@see {@link fail}` (but either seems fine).

Reply:
Done.